cli.js

```diff
@@ -85,7 +92,9 @@ if (typeof flags.language === 'string') {
 	config.set('language', language);
 }
 
-const filepath = `${filename}${extension}`;
+const directory = path.resolve(flags.directory.endsWith(path.sep) ? flags.directory.slice(0, -1) : flags.directory);
```

Comment:
Like I said in a previous comment, you don't need this path separator logic, since `path.join` handles duplicate slashes for us.

Reply:
Makes sense. Apologies for missing these changes.